### ISSUE TITLE
Add array-like type to r and h parameter of get_Bnec (improvement)

### ISSUE THREAD

**Issue by lmar76** [miktoki/swtools#1]
Hi Mikael, why don't add the possibility to set the r and h parameter in the get_Bnec function as an array? Currently it is possible to use only a float. This could be useful when I have to calculate the Bnec from model at the r values taken from the Swarm MAG-L products rather than using a single r or h value per computation.

**Comment by miktoki** [comment]
I agree that it makes sense to define it as an array if it is to compare to measurements.

I added the functionality to define `r` or `h` as an array now. But at least for now, I set it to force the number of values of `r` (or `h`) to be equal to the number of values in `latitude` as I think a 2D radius array might be overkill and might be less intuitive.

You would have to clone the repository so you get v1.2.3 to get the functionality. If this is ok, you can close this issue.

**Comment by lmar76** [comment]
It is ok! Thank you! ...Now I have to understand how to select in the result the same combinations (lat, lon) found in the Swarm product... :-)

**Comment by miktoki** [comment]
I was thinking about the same thing 2 days ago, and looked into it. Right now the whole framework assumes a lat-lon grid, so it is a bit of work to change it, but I can add it eventually

**Comment by lmar76** [comment]
The lat/lon grid is useful for computation (e.g. in case of plots or predictions) but it is a bit tricky to do a comparison with a Swarm product. In some cases could be useful to "restrict" the computation only to the interested lat/lon combinations. However, I don't want to waste too much of your time :)
